library/fedora

```diff
@@ -22,7 +22,7 @@ s390x-Directory: s390x/
 Tags: 33
 Architectures: amd64, arm64v8
```

Comment:
Are ppc64le, and s390x missing here? Same on line 32.

Reply:
yeah I think this is missing :-) @siddharthvipul

Reply:
I don't this this exclusion is new in this PR though, right?  This one's updating the only s390x and ppc64le images that we already have -- should we review this for now and have y'all work this thread separately to figure out whether more arches should be added to these older releases?

Reply:
@tianon Issue is that they are omitted in the first place, they should be there and these are current release.
@cverna :)